client/components/drop-zone/style.scss

```diff
@@ -57,3 +64,26 @@
 .drop-zone.is-full-screen .drop-zone__content {
 	font-size: 30px;
 }
+
+.drop-zone.drop-zone__custom-featured-image {
+	background-color: rgba( $orange-jazzy, 0.7 );
+	border: 6px solid $orange-jazzy;
+
+	margin: 0;
+
+	&.is-dragging-over-element {
+		background-color: rgba( lighten( $orange-jazzy, 20 ), 0.8 );
+	}
+
+	.drop-zone__content {
+		color: #542B06;
+
+	}
+}
+
+.is-hidden {
```

Comment:
Thanks for pointing me to that. The CSS was in the wrong place :) 

### The question about the class name:

The class is set when the `DropZone` is being rendered on the screen. It hides the `editor-drawer-well` content that is in the Featured Image block, because otherwise it shows up behind the dropzone and doesn't look good. 

Comparsion:
![compare-featured-image](https://cloud.githubusercontent.com/assets/184938/25009954/1eaeee10-2071-11e7-873b-b1c06b1dee1c.png)

It could use better naming, because `is-hidden` is very general in meaning, but I don't feel comfortable to naming it `is-dropzone-visible` because it may be used in some other case in the future.